Translators/RoadConditions/main.py

```diff
@@ -0,0 +1,138 @@
+import json
+import requests
+import copy
+import logging
+import os
+from pgquery import query_db
+from tim_generator import get_geometry, get_itis_codes
+from flask import request, Flask
+from datetime import datetime
+
+app = Flask(__name__)
+
+log_level = os.environ.get('LOGGING_LEVEL', 'INFO')
+logging.basicConfig(format='%(levelname)s:%(message)s', level=log_level)
+
+def calculate_direction(coordinates):
+    try:
+        long_dif = coordinates[-1][0] - coordinates[0][0]
+        lat_dif = coordinates[-1][1] - coordinates[0][1]
+    except ValueError as e:
+        return "unknown"
+    except IndexError as e:
+        return "unknown"
+
+    if abs(long_dif) > abs(lat_dif):
+        if long_dif > 0:
+            # eastbound
+            direction = "I"
+        else:
+            # westbound
+            direction = "D"
+    elif lat_dif > 0:
+        # northbound
+        direction = "I"
+    else:
+        # southbound
+        direction = "D"
+    return direction
+
+def translate(rc_geojson):
+    tims = {"timRcList": []}
+
+    for feature in rc_geojson["features"]:
+        if (len(feature["geometry"]["coordinates"]) <= 2):
+            continue
+        tim_body = {}
+        tim_body["clientId"] = feature["properties"]["nameId"].replace("_", "-").replace("/", "-")
+        tim_body["direction"] = calculate_direction(feature['geometry']['coordinates'])
+        tim_body["segment"] = feature["properties"]["routeSegmentIndex"]
+        tim_body["route"] = feature["properties"]["routeName"].replace("_", "-")

```

Comment:
(non-blocking) Using a model class for this 'feature' object would make the code much easier to digest.

For example, this line could be rewritten as the following:
```
tim_body["route"] = feature.get_route_name()
```

Reply:
Added a new class for this